esprit/tasks.py

```diff
@@ -96,6 +96,57 @@ def dump(conn, type, q=None, page_size=1000, limit=None, method="POST", out=None
             record = transform(record)
         out.write(json.dumps(record))
 
+
+def create_alias(conn, alias):
+    actions = raw.to_alias_actions(add=[{"alias": alias, "index": conn.index}])
+    print raw.post_alias(conn, actions).json()
+
+
+def repoint_alias(old_conn, new_conn, alias):
+    actions = raw.to_alias_actions(add=[{"alias": alias, "index": new_conn.index}],
+                                   remove=[{"alias": alias, "index": old_conn.index}])
+    print raw.post_alias(new_conn, actions).json()
```

Comment:
Same comment - will just be a bit odd to run this (if you've not delved into the code behind the function) and get a bunch of JSON output.